pepper-apis/dsm-core/src/main/resources/liquibase/CMI/DDP-7641_set_kit_request_settings_id_to_auto_increment.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
because this is a general change to the schema, can we move it out of the cmi folder please